docs/components/dropdown.md

```diff
@@ -1,8 +1,8 @@
 <script setup>
-import FwbDropdownExamplePlacement from './dropdown/examples/FwbDropdownExamplePlacement.vue'
-import FwbDropdownExampleAlignment from './dropdown/examples/FwbDropdownExampleAlignment.vue'
-import FwbDropdownExampleListGroup from './dropdown/examples/FwbDropdownExampleListGroup.vue'
-import FwbDropdownExampleTrigger from './dropdown/examples/FwbDropdownExampleTrigger.vue'
+import FwbDropdownExamplePlacement from './dropdown/examples/FwbDropdownExamplePlacement.vue';
+import FwbDropdownExampleAlignment from './dropdown/examples/FwbDropdownExampleAlignment.vue';
+import FwbDropdownExampleListGroup from './dropdown/examples/FwbDropdownExampleListGroup.vue';
+import FwbDropdownExampleTrigger from './dropdown/examples/FwbDropdownExampleTrigger.vue';
```

Comment:
Hey,
We're missing here import of `FwbDropdownExampleColors` for the new example to work properly.

Also we're missing `FwbDropdoenExampleCloseInside` import here... and please remove semicolons as we're not using them in the project.

Reply:
The `close inside` example is fixed in #333 

I corrected the other suggestions - thanks for pointing them out! I'll make sure I got the right code style in the future (my IDE suggested these changes, but I'll take extra care that it will be consistent from now on)